--- origsrc/audacious-plugins-3.4.3/acinclude.m4	2014-01-03 18:57:03.000000000 -0600
+++ src/audacious-plugins-3.4.3/acinclude.m4	2014-01-31 03:47:38.698396400 -0600
@@ -110,12 +110,6 @@ if test $HAVE_LINUX = yes ; then
 	LDFLAGS="$LDFLAGS -Wl,-z,defs"
 fi
 
-dnl MinGW needs -march=i686 for atomics
-dnl ===================================
-if test $HAVE_MSWINDOWS = yes ; then
-	CFLAGS="$CFLAGS -march=i686"
-fi
-
 dnl Byte order
 dnl ==========
 AC_C_BIGENDIAN([BIGENDIAN=1], [BIGENDIAN=0],
@@ -186,9 +186,9 @@ AC_PROG_MAKE_SET
 AC_PATH_PROG([RM], [rm])
 AC_PATH_PROG([MV], [mv])
 AC_PATH_PROG([CP], [cp])
-AC_PATH_PROG([AR], [ar])
-AC_PATH_PROG([RANLIB], [ranlib])
-AC_PATH_PROG([WINDRES], [windres])
+AC_PATH_TOOL([AR], [ar])
+AC_PATH_TOOL([RANLIB], [ranlib])
+AC_PATH_TOOL([WINDRES], [windres])
 
 dnl Check for POSIX threads
 dnl =======================
--- origsrc/audacious-plugins-3.4.3/configure.ac	2014-01-03 18:57:03.000000000 -0600
+++ src/audacious-plugins-3.4.3/configure.ac	2014-01-31 03:56:45.121650100 -0600
@@ -1093,7 +1093,7 @@ fi
 
 dnl *** End of all plugin checks ***
 
-plugindir=`pkg-config audacious --variable=plugin_dir`
+plugindir=`$PKG_CONFIG audacious --variable=plugin_dir`
 AC_SUBST(plugindir)
 
 dnl XXX
